automated/linux/network-basic/network-basic.sh

```diff
@@ -50,7 +52,12 @@ run "route" "print-routing-tables"
 run "ip link set lo up" "ip-link-loopback-up"
 run "route" "route-dump-after-ip-link-loopback-up"
 run "ip link set ${INTERFACE} up" "ip-link-interface-up"
-run "ip link set ${INTERFACE} down" "ip-link-interface-down"
+# NFS mounted filesystem hangs when interface set down
+# so skip set <interface> down
+
+if [ "${NFS}" = "false" ]; then
```

Comment:
Could you make it case insensitive:
[ "${NFS}" = "false" ] || [ "${NFS}" = "False" ]

Also please add 'skip' to the 'ip-link-interface-down' if the condition is not met

Reply:
Thanks @mwasilew really valuable comment.
I have addressed your comments.